autopilot/agent_constraints.go

```diff
@@ -100,7 +100,7 @@ func (h *agentConstraints) ChannelBudget(channels []Channel,
 	// present within the set of active channels.
 	var totalChanAllocation btcutil.Amount
 	for _, channel := range channels {
-		totalChanAllocation += channel.Capacity
+		totalChanAllocation += channel.LocalFundedAmt
```

Comment:
Instead of the funded amount, I think we should use the current channel balance?